.github/workflows/on-push.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           echo "NODE_ENV=test" >> .env
           echo "APP_NAME=jira" >> .env
           echo "APP_ID=${{ secrets.E2E_GITHUB_APP_ID }}" >> .env
-          echo "WEBHOOK_SECRET=${{ secrets.E2E_GITHUB_WEBHOOK_SECRET }}" >> .env
+          echo "WEBHOOK_SECRETS=${{ secrets.WEBHOOK_SECRETS }}" >> .env
```

Comment:
With the prefix please.  This webhook is only specific for e2e - please follow the convention.

Reply:
Updated